extensions/MultiUpload/multiupload.js

```diff
@@ -266,10 +266,10 @@ function fillDestFilename( id ) {
 	current = id.replace( /[^0-9]/g, '' );
 	if( wgFileExtensions && id != 'wpUploadFileURL' ) {
 		var found = false;
-		if( fname.lastIndexOf( '.' ) != -1 ) {
-			var ext = fname.substr( fname.lastIndexOf( '.' ) + 1 );
-			for( var i = 0; i < wgFileExtensions.length; i++ ) {
-				if( wgFileExtensions[i].toLowerCase() == ext.toLowerCase() ) {
+		if ( fname.lastIndexOf('.') !== -1 ) {
+			var ext = fname.substr( fname.lastIndexOf('.') + 1 ).toLowerCase();
+			for (var wgfi in wgFileExtensions) {
+				if ( wgFileExtensions[wgfi].toLowerCase() === ext ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ANestedIfDepth)